packages/blade/src/components/Chip/_decisions/decisions.md

```diff
@@ -0,0 +1,144 @@
+# Chip Decisions <!-- omit in toc -->
+
+Chips embody a collection of selectable entities that empower users to engage in choices, sift through content, and invoke pertinent actions.
+
+This document outlines the API details of the `Chip` and `ChipGroup` components, encompassing their structural composition, functional attributes, and visual representation.
+
+<img width="100%" src="./chip-variants.png" alt="Different variants of the Chip component">
+
+- [Design](#design)
+- [`Chip` API](#chip-api)
+- [`ChipGroup` API](#chipgroup-api)
+- [Examples](#examples)
+  - [Basic/Uncontrolled Usage](#basicuncontrolled-usage)
+  - [Controlled Usage](#controlled-usage)
+- [Accessibility Considerations](#accessibility-considerations)
+- [Open Questions](#open-questions)
+
+## Design
+
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?type=design&node-id=52377%3A27686&mode=design&t=pFzCay0x4a6sTcoQ-1) to all variants of the `Chip` component.
+
+## `Chip` API
+
+> **Note**
+>
+> The `Chip` component is inherently tied to the `ChipGroup` and cannot be utilized outside its context.
+
+| Prop                    | Type                                   | Default     | Description                                                                                                                                        | Required |
+| ----------------------- | -------------------------------------- | ----------- | -------------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| **children**            | `React.ReactText \| React.ReactText[]` | `undefined` | Renders the label text for the Chip                                                                                                                | ✅       |
```

Comment:
If we're restricting the users to only pass a string, please add a `textTransform` prop on `Chip` to control the text casing. 

We've had earlier Chip designs that used upper case while the Blade design examples are using capital case.

Reply:
Yes,  `<Chip> {value.toUpperCase()} </Chip>` is allowed.

Reply:
All consumers can write a javascript util for transformText yes but why go to JS for something that can be done with CSS?

Reply:
@snitin315 Can you please add this example in the docs? Generally we'd use CSS for this use-case, not JS.

Reply:
what is the difference between passing it as prop `textTransform: upperCase` vs doing `text.toUpperCase()` inside children?

Reply:
For styling changes we usually go for CSS, doing `string.toUpperCase` is generally not the first solution that comes to mind. In my experience, it's common/expected to get review comments on PRs as well for "if we can do it with CSS, why use JS". It's fine if we want to keep it this way but it'd be nice to have it documented as an example since the API isn't intuitive for the use-case IMO.

They also [behave slightly differently][1] but that's probably an edge case, not the main concern here.

[1]: https://stackoverflow.com/a/53731879/7435656